recipe/make_wrapper.py

```diff
@@ -0,0 +1,31 @@
+import platform
+import os
+from os.path import join
+
+
+PKG_VERSION = os.environ["PKG_VERSION"]
+
+SHELL_CONTENT = """#!/bin/bash
+java -Xmx500M -cp "$CONDA_PREFIX/lib/antlr4-%s-complete.jar:$CLASSPATH" org.antlr.v4.Tool "$@"
```

Comment:
This can be easily done in the command script
```
echo '#!/bin/bash' > $PREFIX/bin/antlr4
echo 'java -Xmx500M -cp "'$PREFIX'/lib/antlr4-'$PKG_VERSION'-complete.jar:$CLASSPATH" org.antlr.v4.Tool "$@"' >> $PREFIX/bin/antlr4
```